library/packages/src/modules/SlideShow.rb

```diff
@@ -778,18 +780,29 @@ def HandleInput(button)
       button = deep_copy(button)
       if button == :showDetails && !ShowingDetails()
         Builtins.y2milestone("User asks to switch to details")
+        @user_switched_to_release_notes = false
         @user_switched_to_details = true
         SwitchToDetailsView()
       elsif button == :showSlide && !ShowingSlide()
         if Slides.HaveSlides
+          if @user_switched_to_release_notes
+            # The user is switching from realease notes to slide show.
+            # In order to not disturbe the user while reading the release notes
+            # we are not updatding the tabs although the slide show has been
+            # changed meanwhile. So we are updating it now before switching to
+            # slide show.
```

Comment:
to be honest I do not get what this comment trying to say. User switched to release notes and now user click on show slide show. So it show release tab and then load slides? I probably missing something.